app/src/sharedTest/java/org/oppia/android/app/topic/info/TopicInfoFragmentTest.kt

```diff
@@ -190,8 +196,7 @@ class TopicInfoFragmentTest {
   }
 
   @Test
-  // TODO(#973): Fix TopicInfoFragmentTest
-  @Ignore
+  @RunOn(TestPlatform.ESPRESSO) // TODO(#1612): Enable for Robolectric.
```

Comment:
This doesn't seem like the correct issue number. Should this be referring to #2057 instead? Ditto below.

Reply:
done